lib/network-layer/src/Cardano/Wallet/Network.hs

```diff
@@ -275,14 +276,20 @@ mapChainFollower fpoint12 fpoint21 ftip fblocks cf =
 -------------------------------------------------------------------------------}
 
 -- | Error while trying to send a transaction
-data ErrPostTx = ErrPostTxValidationError Text | ErrPostTxMempoolFull
+data ErrPostTx
+    = ErrPostTxValidationError Text
+    | ErrPostTxMempoolFull
+    | ErrPostTxEraUnsupported AnyCardanoEra
     deriving (Generic, Show, Eq)
 
 instance ToText ErrPostTx where
     toText = \case
         ErrPostTxValidationError msg -> msg
         ErrPostTxMempoolFull ->
             "mempool was full and refused posted transaction"
+        ErrPostTxEraUnsupported era ->
+            "submitted transaction is not compatible with the current era: "
```

Comment:
I would rephrase that ... it is not supported with Babbage and Conway at this moment. So tx constructed within Alozno, Mary ,... will not be supported by us